rosettanet/2.0/modules/ROOT/pages/rosettanet-connector-examples.adoc

```diff
@@ -0,0 +1,486 @@
+
+. Provide the purchase order request to the Buyer app.
+. Provide the purchase order request response to the Seller app.
+
+=== Provide the Purchase Order Request Document
+
+Use an HTTP POST to the Buyer app `HTTP_InputListener` endpoint to provide the purchase order request to send to the seller.
+
+You can download a https://s3-us-west-2.amazonaws.com/mulesoft-sites-vendorcontent/public-assets/sample-purchase-order-request-content.xml[sample purchase order request]. You can then use any HTTP tool, such as a browser plugin, standalone tool such as PostMan, console tool such as `curl` to POST the data to the Buyer app.
+
+For example, the following `curl` command posts a purchase order request:
+
+`+curl -v -H "Content-Type: application/text" -XPOST --data-binary @sample-purchase-order-request-content.xml http://localhost:8801+`
+
+RosettaNet Connector generates a RosettaNet message based on the purchase order request and sends it to the seller, responding to the HTTP POST operation with a message identifier. In your console, your output should look like this:
+
+[source,java,linenums]
+----
+INFO  ... Seller received action message pMAIhTBMsGzAf/NFx83KBO9nt+T+DV2RNLhwlpNqnXM=0
+INFO  ... Buyer action message pMAIhTBMsGzAf/NFx83KBO9nt+T+DV2RNLhwlpNqnXM=0 completed as SUCCESS
+----
+
+=== Provide the Purchase Order Confirmation Document
+
+Use an HTTP POST to the Seller app `HTTP_InputListener` endpoint to provide the purchase order confirmation to send to the buyer.
+
+You can download a https://s3-us-west-2.amazonaws.com/mulesoft-sites-vendorcontent/public-assets/sample-purchase-order-confirmation-content.xml[sample purchase order confirmation] and then use any HTTP tool to post the data to the Seller app.
+
+For example, the following `curl` command posts a purchase order confirmation:
+
+`curl -v -H "Content-Type: application/text" -XPOST --data-binary @sample-purchase-order-confirmation-content.xml http://localhost:8802`
+
+RosettaNet Connector generates a RosettaNet message based on the purchase order confirmation and sends it to the buyer, responding to the HTTP POST operation with a message identifier. In your console, your output should look like this:
+
+[source,java,linenums]
+----
+INFO  ... Buyer received action message ng7+TalLLPTJZHok4tQSBi8RYZD8IsD9+iB85cubzM=1
+INFO  ... Seller action message sng7+TalLLPTJZHok4tQSBi8RYZD8IsD9+iB85cubzM=1 completed as SUCCESS
+----
+
+The purchase order confirmation action sent by this sample app is only an example. To send a real purchase order confirmation, you must configure the `replyAttributes` on the RosettaNet `send-action` operation with the information provided by when you received the corresponding purchase order request. These `replayAttributes` enable the RosettaNet protocol to distinguish between many concurrent requests.
+
+== See Also
+
+* xref:connectors::introduction/introduction-to-anypoint-connectors.adoc[Introduction to Anypoint Connectors]
```

Comment:
```suggestion
* xref:connectors::introduction/introduction-to-anypoint-connectors.adoc[]
```

Reply:
I will address this when I work on having the rest of the topics match the template.